### DISCUSSION TITLE
Caching when using `--zig` and x86_64-pc-windows-msvc

### DISCUSSION THREAD

**Discussion by adlotsof** [PyO3/maturin#1947] [Q&A] [answered]
When using `maturin build --target x86_64-pc-windows-msvc --zig`, zig is downloading a lot of libraries. I wonder if there is a way to download these before building the target? E.g. when creating the container for ci, id love to already add the headers for this target

![image](https://github.com/PyO3/maturin/assets/6109332/2dc4480d-2010-4fa8-a1e0-21491a614e35)

**Comment by messense** [comment, marked as answer]
> I wonder if there is a way to download these before building the target?

No way ATM in `maturin`, but you can probably achieve that by building a empty crate with `--target x86_64-pc-windows-msvc` when building the Docker image to populate the cache.

**Reply by adlotsof** [reply]
Building an empty crate works fine - but looking at the resulting image, i believe the approach[ described in this discussion](https://github.com/PyO3/maturin/discussions/1953) would also work fine
